packages/react-cosmos/src/plugins/webpack/webpackConfig/userDepsLoader.ts

```diff
@@ -2,16 +2,10 @@ import { detectCosmosConfig } from '../../../config';
 import { DomRendererConfig } from '../../../shared/rendererConfig';
 import { generateUserDepsModule } from '../../../shared/userDeps';
 import { createDomCosmosConfig } from '../cosmosConfig/dom';
+import { loader } from 'webpack';
 
-type WebpackLoaderContext = {
-  async(): (error: Error | null, result: string | Buffer) => unknown;
-  addContextDependency(directory: string): unknown;
-};
-
-module.exports = async function injectUserDeps(source: string) {
+module.exports = function injectUserDeps(this: loader.LoaderContext) {
```

Comment:
My first time having to do _this_ (excuse the pun).

It's a [pseudo parameter](https://www.typescriptlang.org/docs/handbook/functions.html#this-parameters).

Reply:
🆒